indy_node/test/catchup/test_txn_version_controller.py

```diff
@@ -70,3 +70,15 @@ def test_cleanup_after_update_version(txn_version_controller, pool_upgrade_txn):
     # send NODE_UPGRADE for the old version, check that the version didn't change (doesn't has a quorum)
     txn_version_controller.update_version(create_node_upgrade_txn(version1, "Node2"))
     assert txn_version_controller.version == version2
+
+
+def test_get_version_with_timestamp(txn_version_controller, pool_upgrade_txn):
+    version1 = "version1"
+    version2 = "version2"
+
+    txn_version_controller.update_version(create_node_upgrade_txn(version1, "Node1", 10))
+    txn_version_controller.update_version(create_node_upgrade_txn(version2, "Node1", 100))
+
+    assert txn_version_controller.get_pool_version(0) is None
+    assert txn_version_controller.get_pool_version(11) == version1
```

Comment:
What about `txn_version_controller.get_pool_version(10)`? Is it `None` or `version1`?
What about `txn_version_controller.get_pool_version(10)`? Is it `version1` or `version2`?